src/ngraph/runtime/cpu/kernel/relu.hpp

```diff
@@ -45,6 +45,22 @@ namespace ngraph
                     out.device(eigen::global_thread_pool_device) = in0.cwiseMax(ElementType(0));
                 }
 
+                template <typename ElementType>
+                void bounded_relu(void* input0, void* output, ElementType alpha, size_t count)
+                {
+                    Eigen::array<Eigen::Index, 1> out_dims, in_dims;
+
+                    out_dims[0] = in_dims[0] = count;
```

Comment:
Thanks for keeping it consistent. In a future refactor we could just use in_dims in places like this and the cwise macros

Reply:
sounds good.